samcli/local/apigw/local_apigw_service.py

```diff
@@ -690,8 +701,9 @@ def _request_handler(self, **kwargs):
             )
 
             if lambda_authorizer_exception:
-                LOG.error("Lambda authorizer failed to invoke successfully: %s", lambda_authorizer_exception.message)
+                LOG.error("Lambda authorizer failed to invoke successfully: %s", exception_name)
```

Comment:
Is it by design that no return calls are made for this condition?

Reply:
There is a separate condition to return Flask service errors if they were created during the catching of other exceptions in this try block, but we don't re-raise these exceptions since these would likely indicate a user related exception (eg. unauthorized request, invalid response format)